packages/fxa-auth-server/lib/pushbox/index.ts

```diff
@@ -56,15 +57,20 @@ const PUSHBOX_STORE_SCHEMA = isA
 // of helper functions to allow us to store arbitrary
 // JSON-serializable objects.
 
-function encodeForStorage(data) {
+function encodeForStorage(data: any) {
   return base64url.encode(JSON.stringify(data));
 }
 
-function decodeFromStorage(data) {
+function decodeFromStorage(data: string) {
   return JSON.parse(base64url.decode(data));
 }
```

Comment:
Should these two functions be exported?
I saw we also copy-pasta the `encodeForStorage()` bit in `packages/fxa-auth-server/test/remote/pushbox/db.ts#L32` below.

Reply:
I can do that in another PR for the epic.

Reply:
We don't know for sure that the data is a string?